torch/csrc/distributed/c10d/Functional.cpp

```diff
@@ -66,10 +66,11 @@ c10d::ReduceOp to_reduce_op(const std::string& reduce_op) {
   return it->second;
 }
 
-at::Tensor all_reduce_(
-    at::Tensor input,
-    const std::string& reduce_op,
-    const std::string& group_name) {
+at::Tensor& all_reduce_(
+    at::Tensor& input,
+    std::string reduce_op,
+    // c10::string_view group_name,
+    std::string group_name) {
```

Comment:
Out of cuorisity, are these function signature changes necessary for making native c10d_functional ops work with AOTInductor? Thanks.